src/pluginVanilla/java/mcp/mobius/waila/plugin/vanilla/provider/PlantProvider.java

```diff
@@ -58,6 +65,10 @@ public void appendBody(ITooltip tooltip, IBlockAccessor accessor, IPluginConfig
                 addMaturityTooltip(tooltip, accessor.getBlockState().getValue(BlockStateProperties.AGE_3) / 3.0F);
             }
         }
+
+        if (config.getBoolean(Options.CROP_GROWABLE) && accessor.getBlock() != Blocks.COCOA && accessor.getBlock() != Blocks.NETHER_WART) {
+            addGrowableTooltip(tooltip, accessor.getWorld().getRawBrightness(accessor.getPosition(), 0));
+        }
```

Comment:
This assumption might be wrong for modded crops, I think it needs to check if the block is from the `minecraft` namespace.
```java
IModInfo.get(accessor.getBlock()).getId().equals("minecraft");
```